docs-v2/integrations/all/coros.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Coros
+sidebarTitle: Coros
+---
+
+API configuration: [`coros`](https://nango.dev/providers.yaml), [`coros-sandbox`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [OAuth related docs](https://www.dropbox.com/scl/fo/6ps1297tn9pfo7qmcb0o8/AItfHWAW8t-jZ0NIrAaT0hg?e=1&preview=COROS+API+Reference+V2.0.5(Updated).pdf&rlkey=kbq4zmu47j9c3c6qu7b96z39f&st=ntqasbgo&dl=0)
+-   [How Register an Application](https://support.coros.com/hc/en-us/articles/17085887816340-Submit-an-API-Application)
+-   Scopes - Coros does not list any scopes in their documentation
+-   [Oura REST API](https://www.dropbox.com/scl/fo/6ps1297tn9pfo7qmcb0o8/AItfHWAW8t-jZ0NIrAaT0hg?e=1&preview=COROS+API+Reference+V2.0.5(Updated).pdf&rlkey=kbq4zmu47j9c3c6qu7b96z39f&st=ntqasbgo&dl=0)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+- Coros does not list any Oauth scopes in their documentation, so it will be a trial and error process to figure out what is needed.
```

Comment:
same goes for this line, it would also be safe to remove it since its mentioned above. You can also change the location of this line; `Scopes - Coros does not list any scopes in their documentation` to the `API gotchas` section